Quickstart/00-Starter-Seed/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM microsoft/dotnet:latest
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
 
-COPY . /app
-
-WORKDIR /app
+WORKDIR /source
 
+COPY *.csproj .
 RUN ["dotnet", "restore"]
 
+COPY . .
 RUN ["dotnet", "build"]
 
-EXPOSE 3010/tcp
+EXPOSE 5000/tcp
```

Comment:
Any specific reason for changing the port numbers?

Reply:
5000/5001 is the default for a new ASP.NET Core project. It just reduces the diff between the Auth0 template and the usual out of the box experience.

Reply:
I've updated the docker images and returned the host port back to 3010.